pkg/build/edgeimage/internal/kube/source.go

```diff
@@ -17,28 +17,64 @@ limitations under the License.
 package kube
```

Comment:
Is there any difference between the command to install kubeedge using keink now and before? If so, can you show it in the readme file?

Reply:
No difference